Basic-Car-Maintenance-Widget/Basic-Car-Maintenance-WidgetExtension.entitlements

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.developer.applesignin</key>
+	<array>
+		<string></string>
+	</array>
+	<key>keychain-access-groups</key>
+	<array>
+		<string>$(AppIdentifierPrefix)com.mikaelacaron.Shared</string>
```

Comment:
what does this entitlement do? also should this be `.Shared`? as opposed to matching the bundle ID with the app name

and should there be a `.` between `$(AppIdentifierPrefix)` and `com.mikaelacaron.Shared`

Reply:
I can change the `Shared` portion. It was a standard that I saw other devs using online.